content/documentation/multiformat-output.md

```diff
@@ -1,12 +1,222 @@
+2. Update the `baseURL` in `config/site.yml` to match the URL where the site will ultimately be hosted.
+3. Output your site following the directions above.
+4. Upload the contents of the `site` folder to your web host based on the directions they provide.
+
+If you do not already have a web server or hosting plan, we’ve found that [Netlify](https://www.netlify.com) offers a couple excellent options.
+
+### Manual Deploy with Netlify
+
+Manual deploy is ideal if you have a small site or want to run a quick preview. Keep in mind, with this option, you will need to go through the process of rebuilding the site, compressing files, and reuploading them to Netlify each time you make an update, which may be burdensome if you have a lot of images or larger files. For continuous deployment please see [*Continuous Deploy with Netlify*](#continuous-deploy-with-netlify).
+
+1. When you are ready to launch your project, run `quire site` in your command-line shell.
+
+2. Navigate to your project in your home directory and compress the `site` folder.
+
+3. Go to Netlify Drop: [https://app.netlify.com/drop](https://app.netlify.com/drop). Make sure you are logged in to your account and then drag-and-drop your compressed `site` folder into the indicated area.
+
+4. You will be given a default URL to preview your project. Rename this URL by navigating to “Site settings” and changing the site name. (You also have the option to buy a domain or set-up a domain you already own.)
+
+If you make further edits to your project and would like to preview them you will need to repeat this process.
+
+1. Delete the old compressed `site` folder.
+
+2. Run `quire site` again (your files will be automatically overwritten.)
+
+3. Compress the newly updated `site` folder.
+
+4. In Netlify, navigate to “Deploys” at the top of the page. You will see a blank space that reads, “Need to update your site." Simply drag-and-drop your new compressed `site` folder here and your link will be automatically updated.
+
+### Continuous Deploy with Netlify
+
+*This deployment option requires a basic understanding of how to create a repository, and commit and merge changes in GitHub. If you are new to GitHub, we recommend starting with [GitHub Docs](https://docs.github.com/en/github) to learn by topic or checking out the [GitHub for Quire](/documentation/github) section of our documentation. We also encourage you to check out Coding Train’s video series is a great resource as well [Git and Github for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV).*
+
+By keeping your project files on GitHub and linking them directly to your Netlify account, any time you merge changes in Github, your preview link will be automatically updated. This process requires a few extra steps to get set-up, but will save time in the long run. This process also works with GitLab and Bitbucket.
```

Comment:
Let's move "This process also works with GitLab and Bitbucket." up to the first paragraph. Maybe something like:

"This deployment option requires a basic understanding of how to create a repository, and commit and merge changes in GitHub, GitLab, or Bitbucket. We use GitHub and our documentation uses that as an example throughout. If you are new to GitHub, we recommend starting with ..."

FWIW, the reason I'm slightly on it with this GitLab and Bitbucket thing is that I'm increasingly aware of our biases toward certain services and tools—like our preference for GitHub over the others, or Macs over PCs, or Atom over VS Code—and I want to try to mitigate that where we can. Not to the point where it's confusing for Quire users about which tool to go with if they're not sure, but so that users who already are on those other tools won't feel othered.